tests/userId.spec.js

```diff
@@ -1,6 +1,9 @@
-const userId = require("../trials/userId.js");
-const { init } = require("../app.js");
-const jsPsych = require("jspsych-react");
+import { userId } from "../trials/userId.js"
+import { init } from "../app.js"
+import { initJsPsych } from "jspsych"
```

Comment:
There is inconsistency with the use of semi colons at the end of the import statements. In `trials/countdown.js`, you still use semi-colons. Can we use semi-colons for all of them?

Reply:
Sure thing, sounds good.